File: Assignment 50/Example2.java
/* 2. Write a java program which accept number of rows and number of columns from user and display below pattern.
 
Input:  iRow = 4   iCol = 4

Output:   *   *  *  #
          *   *  #  @
          *   #  @  @
          #   @  @  @
 */


 import java.util.*;

class DisplayPattern
{
    public void Pattern(int iRow, int iCol)
    {
        int i=0 ,j=0;
 for(i = 1; i <= iRow; i++)
    {
        for(j = 1; j <= iCol; j++)
        {
            if(j < (iCol - i + 1))
            {
                System.out.print("*\t");
            }
          else if(j ==(iCol - i + 1))
            {
                System.out.print("#\t");
            }
            else
            {
                System.out.print("@\t");
            }
        }
        System.out.println();
    }
    }
}

class Example2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.print("Enter number of rows: ");
        iValue1 = sobj.nextInt();

        System.out.print("Enter number of columns: ");
        iValue2 = sobj.nextInt();

        DisplayPattern dobj = new DisplayPattern();
        dobj.Pattern(iValue1, iValue2);
    }
}

File: Assignment13/Example2.c
// 2. Accept number of rows  and  number of column from usser and display below pattern 

#include<stdio.h>
 
void Pattern(int iRow,int iCol)
{
int i=0 ,j=0;
char ch='A';

for(i=1;i<=iRow;i++)
{
    for(j =0; j <= iCol ; j++)
    {
        if(i%2==0)
        {
            printf("%c",'A'+j);
        }
        else
        {
             printf("%c",'a'+j);

        }
      
    }
    printf("\n");
}
}
 


int main ()

{
    int iValue1=0,iValue2=0;

    printf("Enter number of rows and columns:");
    scanf("%d %d",&iValue1,&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}